synchro/__init__.py

```diff
@@ -618,7 +612,7 @@ def __init__(self, delegate):
         super().__init__(delegate)
 
     def next(self):
-        motor_grid_out = super().next()
+        motor_grid_out = next(super())
```

Comment:
Why not `super().next()`?

Reply:
`B305 `.next()` is not a thing on Python 3. Use the `next()` builtin. For Python 2 compatibility, use `six.next()`.`

Reply:
But Cursor.next() it is a thing in pymongo. That's what's confusing to me. I think we should disable this check as long as we support next() methods.

Reply:
Yeah I'll disable `B305` globally here as well.